src/main/java/com/example/bookstore/dto/BookDto.java

```diff
@@ -1,15 +1,29 @@
 package com.example.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Positive;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class BookDto {
     private Long id;
+    @NotEmpty
     private String title;
+    @NotEmpty
     private String author;
+    @NotNull
+    @Pattern(regexp = "^(978|979)-\\d{1,5}-\\d{2,7}-\\d{1,6}-[\\dX]$")
```

Comment:
what about [this annotation](https://docs.jboss.org/hibernate/stable/validator/api/org/hibernate/validator/constraints/ISBN.html)?